### ISSUE TITLE
Feature: Allow surpressing curl progress bar

### ISSUE THREAD

**Issue by danieldemus** [sdkman/sdkman-cli#982]
When building a docker image on jenkins, that includes installing something via sdkman fills the log output with 100 lines of curl progress bar output of increasing length. What with also running yum/dnf or apt-get, this can result in the log being truncated by jenkins, meking diagnosing later failures more difficult.

It would be useful to have a command-line switch or config value to surpress the '--progress-bar' parameter to curl in sdkman-utils.sh. Something along the lines of:
```
if [[ "${sdkman_surpress_progress}" != 'true' ]]; then
	curl_params+=(''--progress-bar'')
fi
```

**Comment by helpermethod** [comment]
A simple workaround would be to create a `.curlrc` file which contains the `-s` flag, something like

    echo "-s" > ~/.curlrc

**Comment by danieldemus** [comment]
Yeah thanks. That worked. Though I used '--no-progress-bar' to only suppress the progress bar.

**Comment by helpermethod** [comment]
Nice! I'm honestly not sure if it makes sense to include the flag if it's as much work as modifying `.curlrc`. Though I could understand that some users would like to get rid of things like the progress bar.

@marc0der Wdyt?

**Comment by marc0der** [comment]
I agree with you @helpermethod. If it's possible to do this directly with the `.curlrc` it seems like the more pragmatic approach.

However, if we find that many people start asking for this feature, we can reconsider introducing a new config variable.
